---

layout: post
author: yuka
title: Tumor Heterogeneity — a hide-and-seek game
description: A Creative Writing piece on Tumor Heterogeneity
categories: [Genomics, Oncology ]
image: https://source.unsplash.com/nO_XsZ54nl8/1600x1200 

---
<div align="center"><small>Photo by  <a href = "https://unsplash.com/@erikvandijk">Erik van Dijk</a> on Unsplash</small></div>


## Short Summary
 A scientist needs to be a good communicator, and this was my first dive into creative writing on a scientific topic. I have previously written short press releases for Asian Scientist: <a href="https://www.asianscientist.com/author/yukasuzuki/"></a>

---

One of the Thirty-Six Strategems derived from the lessons of historical Chinese battles states,
“ Make a sound in the east, then strike in the west.”
These words could hold some truth with regards to the limited success met by the world’s fight against cancer. What if our true enemies, the cells that drive cancer, have been hiding from our radar when we blast the decoy cancer cells with radiation and harmful drugs? The real enemies could be hiding in the west, a small little corner in the patient’s tumour, poised for a deadlier attack.

Our understanding of our enemy has certainly deepened over the past decade. Advances in sequencing technologies have enabled us to decode the genetic abnormalities in tumour biopsies of cancer patients. A tumour biopsy is a small area of cancerous tissue that is removed often with the use of a special needle. Large cancer genomic consortiums such as The Cancer Genome Atlas have profiled thousands of patients of different cancer types using sequencing technologies. For such studies, one small biopsy is taken from a single region of the patient’s tumour. By zooming into one little bit of tumour DNA, genomic research groups identified cancer-driving genes that are recurrently mutated or expressed at abnormal levels in patients. In parallel, decisions on diagnosis and treatment allocation by oncologists are largely made on a single biopsy taken from the patient as well. The consensus is, if all cells in a tumour are identical, then we will be able to identify all defects that need to be targeted in any part of the tumour.

It has become increasingly evident that different parts of a small biological tissue look morphologically different under the microscope. This observation has already been described by the well-respected pathologist Rudolf Virchow in the 1800s. This could mean that the spatially separated sections in a small tumour tissue may be different genetically as well. In 1976, Peter Nowell introduced the Clonal Evolution theory, describing cancer as an evolutionary process. In his theory, he proposed that tumours are derived from a single cancer cell that acquires genetic defects in a step-wise manner. His theory later became a longstanding cancer biology dogma. However, sobering findings on widespread differences in cancer cells from different parts of kidney tumours have recently challenged the dogma that all cancer cells in a tumour are identical.

The question is, what if one biopsy is not representative of the entire genomic landscape of the tumour? By narrowing down to one region of the tumour, are we missing out important driver genes that are lurking in other regions that we have not sequenced? Ignoring such questions certainly has grave implications to the way we are predicting the clinical outcomes of cancer patients.
Heterogeneity among the different populations of cancer cells within a tumour could contribute to therapeutic failure and metastasis. Like a chaotic battlefield, numerous little armies of cancer cells with different levels of malignancy can be strategically dispersed over the cancerous tissue. Targeting one little army with a targeted drug does not equal to victory since the other armies remain unscathed. Once the coast is deemed clear, the surviving armies crawl out of their trenches and grow in numbers under positive selective pressure. They deliver the final blow by transforming into a highly aggressive and resistant sub-population. This is what we know as a cancer relapse. In addition to therapeutic resistance, diversity in the armies of cancer cells within the same tumour can increase the chances of metastasising to another organ. Depending on the selective pressure, fitter armies survive their voyage to a new site and establish a new settlement.

Many in the research and medical communities, unfortunately, do not feel that heterogeneity is a major contributor to cancer progression. Therefore, they do not see the need to conduct multiple-region sequencing studies. One underlying factor is the cost of sequencing. Imagine collecting metastatic tumours that arise from drug failure, which could double or even triple the cost of sequencing. For the doctors, obtaining multiple biopsies from their patients is a challenging task as taking a biopsy can be stressful and uncomfortable to the patient. But if we take a step back and think about it, we might be missing out a lot of important information about every patient’s tumour. An ideal strategy, in my opinion, consists of mapping all possible hideouts of these cancer subpopulations and a longitudinal surveillance to track emerging cancer cells that were not affected by the treatment.

So how can we achieve this? The answer sometimes lies in the unlikeliest places. Recent reports have suggested ‘liquid biopsies’ as an alternative to single tumour biopsies.
Liquid biopsies could be tumour cells or tumour DNA that are shed by the tumour cells into the bloodstream. A small number of studies have observed that this strategy does not require sequencing of multiple regions of the same tumour. Unlike solid tumour biopsies that hold snippets of information of specific small armies of the enemy, these ‘liquid biopsies’ hold the complete information of the entire cohort. By teasing out the cancer cells from other cell debris in the liquid sample, we can dive deep into the malignant genomes of these cancer cells with an extremely sensitive sequencing technology. From the data, we can decode the full repertoire of genetic abnormalities in the patient’s tumour and understand the extent of heterogeneity within the patient’s tumour. From here, we would have a better idea to predict if this patient will fail therapy or metastasise.

The point of translational research is to translate biological findings into relevant guidelines that can improve clinical decisions. The tumour heterogeneity problem is undeniably a prevalent obstacle to getting that vital piece of information. We should stop getting ourselves fooled by our foe and re-examine our research strategies.
